src/identity_tree/initializer.rs

```diff
@@ -78,7 +90,7 @@ impl TreeInitializer {
 
         let mined_items = dedup_tree_updates(mined_items);
 
-        if !self.config.force_cache_purge {
+        if !force_cache_purge {
```

Comment:
this is still the same value? We can just use `self.config.force_cache_purge` no?

Reply:
For second call here https://github.com/worldcoin/signup-sequencer/pull/815/files#diff-564578c6f713fb19e94efd82c09256f2b2f8c831854818c48a8b2b260524a30dR69 it is set to static true to force cache purge.